x11-themes/terminology-themes/terminology-themes-9999.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Color schemes for the Terminology terminal emulator"
+HOMEPAGE="https://charlesmilette.net/terminology-themes/"
+SRC_URI=""
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/sylveon/terminology-themes"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/efl"
+RDEPEND="
+	${DEPEND}
+	x11-terms/terminology"
+
+src_install() {
```

Comment:
the `src_install()` phase here is extraneous. please remove it altogether.

Reply:
I had that first. They were using `PREFIX` though instead of `DESTDIR` in their `Makefile`, so I had to work around it. See https://github.com/sylveon/terminology-themes/blob/master/Makefile#L21

Reply:
I have filed a PR to upstream to change it.